assets/src/data/content/writers/html.ts

```diff
@@ -61,31 +62,17 @@ export class HtmlParser implements WriterImpl {
       .reduce((acc, mark) => `<${mark}>${acc}</${mark}>`, text);
   }
 
```

Comment:
`displayClass` was used to float images/etc left and right, and I left the code there in case we decided to add it back in, but it's probably time to just remove it.